core/src/templates/components/global-footer/global-footer.twig

```diff
@@ -20,24 +25,24 @@
     <div class="su-global-footer__content">
       <nav aria-label="global footer menu">
         <ul class="su-global-footer__menu su-global-footer__menu--global">
-          <li><a href="https://www.stanford.edu">Stanford Home</a></li>
-          <li><a href="https://visit.stanford.edu/plan/">Maps &amp; Directions</a></li>
-          <li><a href="https://www.stanford.edu/search/">Search Stanford</a></li>
-          <li><a href="https://emergency.stanford.edu">Emergency Info</a></li>
+          <li><a href="https://www.stanford.edu">Stanford Home {{ external_link_text }}</a></li>
```

Comment:
```suggestion
          <li><a href="https://www.stanford.edu">Stanford Home{{ external_link_text }}</a></li>
```

Remove the whitespace created by the variable so that they don't show up in the link. Add the whitespace to the variable instead.